### DISCUSSION TITLE
What is the areal scale for net longwave radiation assumed by the SOLWEIG model?

### DISCUSSION THREAD

**Discussion by rarygit** [UMEP-dev/UMEP#234] [Q&A]
I have coupled SOLWEIG with an SPAC model to simulate and compare point of interest (POI) and polygon of interest (POLI) input/ouput that SUEWS cannot provide.

SOLWEIG POI output is either input for, or compared with, the SPAC model POI.

SOLWEIG variables of interest are:
 - Incoming longwave radiation (ldown)
 - Incoming shortwave radiation
 - Surface net radiation [sum of (kdown - kup) + (ldown - lup)] 
 - Outgoing shortwave and longwave radiation (kup, lup)
 - Soil surface temperature (Tg)
 - Corresponding soil emissivity and albedo values

Two questions: 

What information is there regarding the areal scale for net longwave radiation that is assumed by the SOLWEIG model?   

Is the algebra above correct for calculating the surface net radiation? (see https://www.myroms.org/forum/viewtopic.php?t=5286)

**Comment by biglimp** [comment]
Interesting. I hope I can give the answers you need.

- The areal scale of any radiation flux (short or long) is calculated at pixel scale, i.e. the spatial resolution of the input data is e.g. 1 meter for the test dataset from Gothenburg. Regarding the vertical scale, the fluxes is calculated at 1.1 meter above ground level (center of gravity of a standing person)
- Yes,  that is the corrrect way to calculate Q*

I am not familiar with the SPAC model, but if it is a local scale model you can spatially aggregate output from SOLWEIG. However, you should note that radiation fluxes att roof top level are not as sophisticated as those calculated on the ground. Also, we are working on implementing a sort of 1D version of SOLWEIG into SUEWS called BEERS. Maybe that will be useful for you later on.

**Reply by rarygit** [reply]
Thanks for that and very helpful!

The SPAC model is a micro-scale (1 m²) 1D point model. 

I needed a number of criteria to be satisfied for modelling urban extreme weather risk management (IPCC) using stormwater/rainwater green infrastructure networks.

Although there are several models available to simulate urban heat island effects and possible mitigation strategies, very few could be considered applicable to the coupling of rainwater green infrastructure networks with urban extreme weather risk management.

Such risk management requires the capacity of a SPAC model to simulate both water and energy balances, structural soils (gravels, large rocks), water quality (heavy metal retention, biochar remediation), snow pack development, vegetation and root stress (including drought, oxygen deficiency, road salt impacts), runon, subsurface drip irrigation (esp. greywater), as well as vegetation growth, C-N-P-K chemistry and the role of soil micro-organisms/mycorrhiza. Runoff and subsurface pipe drainage is coupled with the SWMM hydraulic network (pipe routing). 

With support from the developer, I am using the SPAC model in an urban context, with the help of SOLWEIG.

We can discuss this further if you like by email.
